packages/scoutgame/src/builders/getDeveloperInfo.ts

```diff
@@ -4,18 +4,17 @@ import { prisma } from '@charmverse/core/prisma-client';
 import { getCurrentSeasonStart, getCurrentWeek } from '@packages/dates/utils';
 import { DateTime } from 'luxon';
 
-import { convertCostToPoints } from '../builderNfts/utils';
-
 import { normalizeLast14DaysRank } from './utils/normalizeLast14DaysRank';
 
 export type DeveloperInfo = {
+  id: string;
   path: string;
   displayName: string;
   avatar: string;
   firstContributionDate: Date;
   level: number;
   estimatedPayout: number;
-  price: number;
+  price: bigint;
```

Comment:
i am not sure how this is working... but this data is used on the frontend, we can't pass bigint over the server?